integration_tests/test_domestic_migration.py

```diff
@@ -289,9 +289,12 @@ def test_pumas_exist_in_states(sim, populations, state_id_col, puma_col):
     ],
 )
 def test_addresses_during_moves(
-    simulants_on_adjacent_timesteps, unit_id_col, address_id_col, state_id_col, puma_col
+    sim, simulants_on_adjacent_timesteps, unit_id_col, address_id_col, state_id_col, puma_col
 ):
     """Check that unit (household and business) address details change after a move."""
+    number_of_states_in_artifact = (
```

Comment:
Is it not possible to get this information directly from the configuration? This way will work, but I imagine it may be a bit slow.